articles/fleet-4.65.0.md

```diff
@@ -1,9 +1,5 @@
 # Fleet 4.65.0 | GitOps mode, automatically install software, certificates in host vitals
 
-<div purpose="embedded-content">
-   <iframe src="https://www.youtube.com/embed/z7pbLb9kp1c?si=xPhedv5F46oR8YR-" frameborder="0" allowfullscreen></iframe>
-</div>
```

Comment:
@lukeheath do you know why we're removing the 4.65 sprint demos video here? (Wondering if this was an unintentional change)